paddle/fluid/pybind/tensor.cc

```diff
@@ -393,11 +393,11 @@ void BindTensor(pybind11::module &m) {  // NOLINT
         Examples:
             .. code-block:: python
 
-                import paddle.base as base
-                import numpy as np
+                >>> import paddle.base as base
+                >>> import numpy as np
 
-                t = base.Tensor()
-                t.set(np.ndarray([5, 30]), base.CPUPlace())
+                >>> t = base.Tensor()
+                >>> t.set(np.ndarray([5, 30]), base.CPUPlace())
```

Comment:
```suggestion
                >>> t = paddle.Tensor()
                >>> t.set(np.ndarray([5, 30]), paddle.CPUPlace())
```

非必要不要用 base（原 fluid）了吧